### ISSUE TITLE
ASE shows blank white screen and no error message when trying to log into Azure Subscription

### ISSUE THREAD

**Issue by beroder** [microsoft/AzureStorageExplorer#657]
**Storage Explorer Version**: 1.4.1
**Platform/OS Version**: Windows 10
**Architecture**: ia32
**Regression From**: No

#### Steps to Reproduce: ####
•	Whenever cx is trying to log in to Azure Storage Explorer 
o	Manage accounts > Add an account > Add an Azure Account > Sign in > enter his credentials 
	There are 2 windows displayed 
•	One saying “Waiting for authentication…” ( screenshot 1)
•	Another one showing just a white blank screen  ( screenshot 2)
	And he is not able to log in, it just stays in this state. 
 
o	Never worked for them to log in to ASE with an Azure Account
o	Only works if they use account name & key

#### Expected Experience: ####
Log in successfully or provide a helpful error message.

#### Actual Experience: ####
Blank white login screen and no error message.

**Comment by daroost** [comment]
Having the same issue with Mac

**Storage Explorer Version**: 1.4.2
**Platform/OS Version**: macOS High Sierra - 10.13.6 
**Architecture**: x64

#### Other note: ####
With developer tools open with the sign in popup, I noticed a 401 error code when calling my organization login portal https://sso.company.com/adfs/ls/wia?client-request-id=.....

**Comment by fralik** [comment]
The same issue on Linux

**Storage Explorer Version**: 1.5.0
**Architecture**: x64
**Build number**: 20181026.3
**Commit**: a9422b7d
**Support Id**: a37ec1cb-aebd-e28c-9e8a-097332a7ff6e

**Other notes**
The blank script appears after redirect to organization's authentication page. Seems like the redirect never comes through.

**Comment by MRayermannMSFT** [comment]
Hello to anyone watching this issue! In 1.7.0 we're introducing the ability to optionally sign-in via device code flow, or as you may be more familiar with, copying a code from Storage Explorer into a page in your browser, and then signing in via your browser.

We're currently previewing that feature in our insiders channel. If y'all are intereseted in trying it early, simply go to Help -> Opt-in To Insider Builds.
![image](https://user-images.githubusercontent.com/21112697/52584072-04563f80-2de6-11e9-90c1-094e4bbcd0b4.png)
Then, go to Help -> Check for updates, wait for 1.7.0 to appear, and install it. Then once you install it, you can enable device code flow by going to Experimental -> Use Device Code Flow Sign In. 
![image](https://user-images.githubusercontent.com/21112697/52584173-3e274600-2de6-11e9-85af-6ed3cf5ed006.png)

And then let us know if things work out for y'all! This issue of blank sign in dialog is one we've been fighting for the longest time, and we're hoping this method of sign-in will unblock y'all. 😄

**Comment by amclin** [comment]
I can replicate this on OSX. I have no trouble logging in to Azure Storage tools on VSCode

**Comment by EggsBenedict** [comment]
FYI, this issue is still present in V1.11.0.  On a OS X the "Use device code flow sign-in" is under "Microsoft Azure Storage Explorer > Settings".

**Comment by MRayermannMSFT** [comment]
Are any of you still having problems? Please upgrade to the latest Storage Explorer and let us know if your issue still occurs. Additionally, if you're having sign in issues, we encourage you to try enabling MSAL. To do so, go to settings -> application -> sign in-> enable Microsoft Authentication Library (MSAL), then restart and try signing in.

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. Please open a new issue if you're still having problems. Thanks.

**Comment by mikeyrcamp** [comment]
I hit this exact same issue on MacOS Catalina today. The MSAL option and "Use device code flow sign-in" both function great. Thanks!

Version Info:
Version: 1.12.0
AzCopy Version: 10.3.3

Platform: macOS
Architecture: x64 
Build Number: 20200116.10
